lib/generators/templates/base_service.rb.erb

```diff
@@ -1,17 +1,21 @@
-class <%= class_name %>Service < Wor::Requests::Base
-  # An example of how to get all repositories for the specified user in github.com
-  # def repositories(username)
-  #   get(
-  #     attempting_to: "get repositories of #{username}",
-  #     path: "/users/#{username}/repos"
-  #   )
-  # rescue Wor::Requests::RequestError => e
-  #   puts e.message
-  # end
+<% if @module_name.present? %>module <%= @module_name.camelize %>
```

Comment:
Instead of asking in the module is present in every line, can we use something like this?:
```ruby
<% if @module_name.present? %>
  module <%= @module_name.camelize %>
    class <%= class_name %> Service < Wor::Requests::Base
      # An example of how to get all repositories for the specified user in github.com
      # def repositories(username)
      #   get(
      #     attempting_to: "get repositories of #{username}",
      #     path: "/users/#{username}/repos"
      #   )
      # rescue Wor::Requests::RequestError => e
      #   puts e.message
      # end

      # def base_url
      #   'https://api.github.com'
      # end
    end
  end
<% end %>
```

Reply:
Mmm i don't like add a big if...else, but is better
```
<% if @module_name.present? %>
module <%= @module_name.camelize %>
  class <%= class_name %>Service < Wor::Requests::Base
    # An example of how to get all repositories for the specified user in github.com
    # def repositories(username)
    #   get(
    #     attempting_to: "get repositories of #{username}",
    #     path: "/users/#{username}/repos"
    #   )
    # rescue Wor::Requests::RequestError => e
    #   puts e.message
    # end

    # def base_url
    #   'https://api.github.com'
    # end
  end
end
<% else %>
class <%= class_name %>Service < Wor::Requests::Base
  # An example of how to get all repositories for the specified user in github.com
  # def repositories(username)
  #   get(
  #     attempting_to: "get repositories of #{username}",
  #     path: "/users/#{username}/repos"
  #   )
  # rescue Wor::Requests::RequestError => e
  #   puts e.message
  # end

  # def base_url
  #   'https://api.github.com'
  # end
end
<% end %>
```